WordPress/wporg-main-2022
# Add .phpunit.result.cache to .gitignore

When you run a PHP unit test locally, the `source/wp-content/tests/phpunit/.phpunit.result.cache` file is generated. This file is not excluded from git, so an unintended file difference will occur.

This file is also excluded in [WordPress core](https://github.com/WordPress/wordpress-develop/blob/d859cbe79092b52dda5fa70daf1385f82f268a2a/.gitignore#L9) and [Gutenberg](https://github.com/WordPress/gutenberg/blob/3ad117e075dc6bab30a40d7f39e5455b5b66748b/.gitignore#L9).

### How to test the changes in this Pull Request:

- Checkout ths branch.
- Run `yarn test:php`.
- Confirm that the `source/wp-content/tests/phpunit/.phpunit.result.cache` file has been generated.
- Run `git status`
- There should be no files to commit.
